code/modules/mob/mob_defines.dm

```diff
@@ -110,3 +110,8 @@
 	var/datum/click_intercept
 
 	var/registered_z
+//sunset start
+	var/adminfrozen = 0 //handle for someone's sleeping time from before they got frozen
```

Comment:
`FALSE`, not `0`

Reply:
not a boolean,its the amount of time they were sleeping before the freeze

Reply:
Oh.